modules/admin_manual/pages/appliance/configuration/wnd_setup.adoc

```diff
@@ -1,4 +1,4 @@
-= Windows Network Drive Configuration (WND)
+= WND in the Appliance (WND)
```

Comment:
```suggestion
= Windows Network Drive in the Appliance (WND)
```

The acronym is already mentioned at the end of the title. If you use the acronym at the start also, then the meaning of "WND" does not appear in the document.

Reply:
My worry is the character limitation in the search window. I want the title to be very distinct from the WND one and from a future document WND Quick Guide.

So we have Configuring Windows Network Drives, WND in the Appliance and WND quick guide.

Reply:
I'm confused now. Either write it out and add the abbreviation at the end, or just abbreviate in the front and neglect the abbreviation at the end. But don't abbreviate in the front and at the end in parenthesis, that's redundant.

If there is some character limit in the search, I vote for these three article headlines:
- Windows Network Drive (WND)
- WND Quick Guide
- WND in the Appliance

Reply:
valid point. I have forgotten to remove the abbreviation at the end.